dotnet/samples/ApplicationInsightsExample/ApplicationInsightsExample.csproj

```diff
@@ -19,10 +19,8 @@
   </ItemGroup>
   
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Extensions\Planning.SequentialPlanner\Planning.SequentialPlanner.csproj" />
-    <ProjectReference Include="..\..\src\Extensions\Planning.ActionPlanner\Planning.ActionPlanner.csproj" />
-    <ProjectReference Include="..\..\src\Extensions\Planning.StepwisePlanner\Planning.StepwisePlanner.csproj" />
     <ProjectReference Include="..\..\src\Connectors\Connectors.AI.OpenAI\Connectors.AI.OpenAI.csproj" />
+    <ProjectReference Include="..\..\src\Extensions\Planning\Planning.csproj" />
```

Comment:
What will we do with future planners we don't want part of this package? Should we follow the `Plugins` pattern and have `Extensions\Planning.Core\Planning.Core.csproj` and later `extensions\Planning.NewPlanner\Planning.NewPlanner.csproj`

Reply:
That's a good question. If we envision needing separate packages for planners still, we could create a `Planning.Core` project for just the shared classes, and add these planners to another project e.g. `Planning.Planners` and one-off/experimental planners could go in a separate project like `Planning.ExperimentalPlanner`.

Reply:
Consistency with Plugins.Core makes sense to me. I renamed this project to Planning.Core. One-off planners that shouldn't be part of the core planning project can follow the pattern you provided.

Reply:
cc @shawncal  too on their thoughts.

Reply:
Talked to Lee/Shawn/Mark offline and the decision was to:

Name the nuget package: **Microsoft.SemanticKernel.Planners.Core**
Add the project under: **./planners/Planners.Core.csproj**
Name the root namespace: **Microsoft.SemanticKernel.Planners**